common/unsafe/src/test/java/org/apache/spark/unsafe/hash/Murmur3_x86_32Suite.java

```diff
@@ -51,6 +51,22 @@ public void testKnownLongInputs() {
     Assert.assertEquals(-2106506049, hasher.hashLong(Long.MAX_VALUE));
   }
 
+  @Test
```

Comment:
It would be good to add JIRA number with a short description as a comment (e.g. `SPARK-23381 ...`)